breathe/renderer/sphinxrenderer.py

```diff
@@ -1455,13 +1455,15 @@ def visit_typedef(self, node) -> List[Node]:
         names = self.get_qualification()
         names.append(node.get_name())
         name = self.join_nested_name(names)
-        if node.definition.startswith('typedef '):
-            declaration = ' '.join([type_, name, node.get_argsstring()])
-        elif node.definition.startswith('using '):
+        if node.definition.startswith('using '):
             # TODO: looks like Doxygen does not generate the proper XML
-            #       for the template paramter list
+            #       for the template parameter list
             declaration = self.create_template_prefix(node)
             declaration += ' ' + name + " = " + type_
+        else:
+            if not node.definition.startswith('typedef '):
+                print("\"@typedef <name>\" found, falling back to typedef")
```

Comment:
Perhaps printing the received definition here is helpful for when people run into this problem. As in appending something like `: {}".format(node.definition)`.

Thoughts?

Reply:
> Perhaps printing the received definition here is helpful for when people run into this problem. As in appending something like `: {}".format(node.definition)`.
> 
> Thoughts?

Can do. I was more inclined to not print anything! And the reason for it, is that on our documentation build systems we actually check warnings; so this new warning, whatever it's message ends up being, would be an extra thing to deal with.

Reply:
Honestly I am doubtful too about this one, perhaps it would also be fine to only have some `TODO`-ish comment in the sources explaining the hacky situation a bit. It's just that the issue still feels a bit uncertain making it hard to really say this is a good, proper fix and not more of a workaround and being silent when odd things are happening might not be a good idea.

Reply:
I changed it to a `TODO`, if required I can bring the warning back!